### ISSUE TITLE
Prerelease-7: Fatal error when deleting a group and not unlinking it from a user

### ISSUE THREAD

**Issue by JohanvdWest** [happner/happner-2#94]
```
SystemError: Error: Find failed on path: /_SYSTEM/_SECURITY/_GROUP/device_chiselcap_SkbBHkSiN
```

The safest option would be to unlink a group from all users when deleteGroup is called.

**Comment by southbite** [comment]
Hi Johan - you have any idea of where/when this is happening, are you getting a stacktrace?

**Comment by southbite** [comment]
ah - its fine - found the issue, think this didnt find its way into your current beta, going to include it with the current PR as it is urgent.

**Comment by southbite** [comment]
Fixed in release 7.1.1
